src/gcp/cloudfunctions.ts

```diff
@@ -426,7 +426,13 @@ export async function deleteFunction(options: any): Promise<Operation> {
  * @param projectId the Id of the project to check.
  * @param region the region to check in.
  */
-export async function listFunctions(projectId: string, region: string): Promise<CloudFunction[]> {
+export async function listFunctions(
```

Comment:
I don't see how it's possible to get unreachable regions when region isn't `-`. Should we use a common helper method and then make `listFunctions` return just a `CloudFunction[]`?

Reply:
Good point, I moved this into a helper method and changed the signature of the exported method. We currently don't use listFunctions anywhere, but it seems like a usual building block to have around, so i'm leaving it for now